app/code/community/Reverb/ReverbSync/Helper/Data.php

```diff
@@ -281,6 +281,7 @@ protected function _getCurlResource($url, $options_array = array())
         $options_array[CURLOPT_SSL_VERIFYPEER] = 0;
         $options_array[CURLOPT_HEADER] = 0;
         $options_array[CURLOPT_RETURNTRANSFER] = 1;
+        $options_array[CURLOPT_USERAGENT] = 'Reverb-Magento MagentoVersion=community-1.9';
```

Comment:
Currently the request is now looking as such

GET /api/my/listings?state=all&sku=new_curl HTTP/1.1
User-Agent: Reverb-Magento MagentoVersion=Community-1.9.2.0 MagentoDomain=reverb_magento.com
Host: sandbox.reverb.com
Accept: */*
X-Auth-Token: 1ea238014d948d056f189978d7ab837d1c8152112dfcf0d2eca2b2cfd53f7d3c
Content-type: application/hal+json
X-Magento-Version: Community-1.9.2.0
X-Magento-Domain: reverb_magento.com

Does this look correct?